### PR TITLE
Windows 10 support - toast notifications

### PR DESCRIPTION
**Pull request by voltagex**
I was unable to run lint.sh on Windows, but I've run the following checks and I believe I'm all good.

```
go vet github.com/variadico/noti/...
golint github.com/variadico/noti/...
gofmt -d .
gofmt -s -d .
```

#### Description
Adds very basic Windows 10 support. Speech and PID polling are unsupported
 
#### Resolves
Ability to use noti on Windows. #36 is not yet fixed but it's a good step.

### CODE REVIEW

**Comment by variadico** [comment]
@voltagex, can you rebase against latest `dev`?

And also run?
```
cd noti
go fmt ./...
```

Finally got a Windows 10 machine to this this on!

File: banner/banner_windows.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// +build windows
+
+package banner
+
+import (
+	"github.com/variadico/noti"
+	toast "github.com/jacobmarshall/go-toast"
+    "log"
```
Comment (variadico) [inline banner/banner_windows.go]: `go fmt ./...` will fix this bad formatting.

File: banner/banner_windows.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+// Notify displays a Windows 10 Toast Notification
+func Notify(n noti.Params) error {
+	 notification := toast.Notification{
+        AppID: "noti",
+        Title: n.Title,
+        Message: n.Message,
+        Icon: "",
+        Actions: nil}
+
+    err := notification.Push()
```
Comment (variadico) [inline banner/banner_windows.go]: Let's just

```
return notification.Push()
```

instead.

**Comment by variadico** [comment]
Tested it. Works. 👍 

Just need to fix those two comments I made and it'll be good to merge.

**Comment by voltagex** [comment]
This is embarrassing - I've fixed the two issues you pointed out (even though I'm sure I ran go fmt before) but I can't work out how to rebase onto dev.

**Comment by variadico** [comment]
Cool. Just merged to `dev`.

https://github.com/variadico/noti/commit/175b0ec7a9def133f0a122f5324f97419f4c0f0c

Thank youuuuUU!!!

**Comment by variadico** [comment]
(I checkout your repo and did the rebasing for you. It _was_ kinda tricky.)

**Pull request closed without merging**